pkg/pgmodel/migrate.go

```diff
@@ -223,9 +239,19 @@ func ensureVersionTable(db *pgxpool.Pool) error {
 	return nil
 }
 
-func getDBVersion(db *pgxpool.Pool) (semver.Version, error) {
+func getSchemaVersion(db *pgxpool.Pool) (semver.Version, error) {
+	var version semver.Version
+	conn, err := db.Acquire(context.Background())
+	if err != nil {
+		return version, err
+	}
+	defer conn.Release()
+	return getSchemaVersionOnConnection(context.Background(), conn.Conn())
```

Comment:
IMO context does not need to start from here since this is not a dependent parent. This would have been the case if a cancel was being called in some part of this function, which is not the case.

Reply:
yeah, we should really be more principled about how we pipe through contexts, and will probably go through and clean them after 0.1.0